src/css/common/buttons.css

```diff
@@ -0,0 +1,37 @@
+.btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 1.714;
+  padding: 14px 87px;
```

Comment:
do we actually need so big paddings as 87px?
Maybe in that place that you inspected there is a button just on full container width?
Because in lessons cards buttons have 16px 39px paddings?

Reply:
agree. took values from figma. changed it to padding: 14px 32px; for mobile and padding: 16px 40px; for desktop+